toast-tk-runtime/src/main/java/io/toast/tk/runtime/AbstractTestPlanRunner.java

```diff
@@ -32,37 +33,37 @@
 
 	private String db;
 
-	protected AbstractProjectRunner() {
+	protected AbstractTestPlanRunner() {
 		super();
 		this.projectHtmlReportGenerator = injector.getInstance(IProjectHtmlReportGenerator.class);
 		this.htmlReportGenerator = injector.getInstance(IHTMLReportGenerator.class);
 	}
 
-	protected AbstractProjectRunner(final Module extraModule, final String host, final int port, final String db) {
+	protected AbstractTestPlanRunner(final Module extraModule, final String host, final int port, final String db) {
 		this(extraModule);
 		this.mongoDbHost = host;
 		this.mongoDbPort = port;
 		this.db = db;
 	}
 
-	protected AbstractProjectRunner(final String host, final int port, final String db) {
+	protected AbstractTestPlanRunner(final String host, final int port, final String db) {
 		this();
 		this.mongoDbHost = host;
 		this.mongoDbPort = port;
 		this.db = db;
 	}
 
-	public AbstractProjectRunner(final Module extraModule) {
-		super(extraModule);
+	public AbstractTestPlanRunner(final Module... extraModules) {
+		super(extraModules);
 		this.projectHtmlReportGenerator = injector.getInstance(IProjectHtmlReportGenerator.class);
 		this.htmlReportGenerator = injector.getInstance(IHTMLReportGenerator.class);
 	}
 
-	public final void test(ITestPlan project, boolean useRemoteRepository) throws Exception {
-		execute(project, useRemoteRepository);
+	public final void test(ITestPlan testplan, boolean useRemoteRepository) throws IOException {
+		execute(testplan, useRemoteRepository);
 	}
 
-	public final void test(final String name, final String idProject, final boolean useRemoteRepository) throws Exception {
+	public final void test(final String name, final String idProject, final boolean useRemoteRepository) throws IOException, IllegalAccessException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to throw at most one checked exception instead of: java.io.IOException, java.lang.IllegalAccessException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1160)